lldap_wrapper.py

```
@@ -0,0 +1,117 @@
+                    "User-Agent": "Python lldap-request",
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                    "Origin": authelia_url,
+                    "Referer": f"{authelia_url}/reset-password/step1",
+                },
+                timeout=10,
+            )
+            if response.status_code not in {200, 204}:
+                return (
+                    False,
+                    f"User created but Authelia reset failed: {response.status_code} - {response.text}",
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)